tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,59 @@
+"""Work with inventory."""
+
+
+def display_inventory(inventory):
+    """Print inventory."""
+    print('Inventory:')
+    items = 0
```

Comment:
I would rather rename it to `items_count` or something similar. Name `items` suggests that it is some kind of collection (list, set, etc.).